hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -56,17 +56,25 @@ if {$FOUNDRY_ROOT != ""} {
 set ri_create_outputs_in_create_env true
 set ri_print_module_nand2_counts true
 # enable analysis of large arrays
-set ri_max_total_range_bits 100000
+set ri_max_total_range_bits 20 
+set ri_prefer_liberty true
+
+############################
+## Apply Liberty File ##
+
+read_liberty /workspaces/repo/hw/top_earlgrey/cdc/ast.lib -db_dir ./RI_compiled_libs/verix_2019_a_p27_2_2022_03_20
```

Comment:
we might need to massage this path a bit to be related to the project root.

Reply:
Yup!